code/modules/mob/mob.dm

```diff
@@ -19,6 +21,17 @@
 	else
 		living_mob_list += src
 	prepare_huds()
+
+	// SPESS BACKGROUND
+	src.spessbg = new /obj/screen()
```

Comment:
`src.` is implied.

This shouldn't be in /mob/New(), it should be in `code/_onclick/hud/hud.dm`, under `/datum/hud/proc/instantiate()`